docs/redirects.csv

```diff
@@ -158,5 +158,10 @@ https://help.expensify.com/articles/expensify-classic/workspaces/Budgets,https:/
 https://help.expensify.com/articles/expensify-classic/workspaces/Categories,https://help.expensify.com/articles/expensify-classic/workspaces/Create-categories
 https://help.expensify.com/articles/expensify-classic/workspaces/Tags,https://help.expensify.com/articles/expensify-classic/workspaces/Create-tags
 https://help.expensify.com/expensify-classic/hubs/manage-employees-and-report-approvals,https://help.expensify.com/articles/expensify-classic/copilots-and-delegates/Approval-Workflows
+https://help.expensify.com/articles/expensify-classic/expenses/expenses/Add-an-expense,https://help.expensify.com/articles/expensify-classic/expenses/Add-an-expense
+https://help.expensify.com/articles/expensify-classic/expenses/expenses/Apply-Tax,https://help.expensify.com/articles/expensify-classic/expenses/Apply-Tax
+https://help.expensify.com/articles/expensify-classic/expenses/expenses/Merge-expenses,https://help.expensify.com/articles/expensify-classic/expenses/Merge-expenses
+https://help.expensify.com/articles/expensify-classic/expenses/reports/Reimbursements,https://help.expensify.com/articles/expensify-classic/bank-accounts-and-payments/Reimbursements
+https://help.expensify.com/articles/new-expensify/expenses/Referral-Program,https://help.expensify.com/articles/expensify-classic/expensify-partner-program/Referral-Program
```

Comment:
Just want to double check that we're intentionally moving an article that references NewDot from a `new-expensify` folder to an `expensify-classic` folder. Does it matter?

Reply:
Good question! So, we’re doing that because there is no Partner Program folder in the NewExpensify hub. This article needs to live in the Partner Program folder, so it will have to live under the Expensify Classic hub.

Reply:
Okay! Just wanted to be sure. Thanks for the additional context 👍

Reply:
Of course! Thanks for asking!